doc/api/events.md

```diff
@@ -278,6 +278,11 @@ emitter.once('event', () => {
 });
 ```
 
+The emitted warning can be inspected with [`process.on('warning')`][] and will
+have the additional `emitter`, `type` and `count` properties, referring to
+the event emitter instance, the event’s name and the number of attached
+listeners, respectively.
+
```

Comment:
May be worth also reiterating that --trace-warnings will print out the stack trace detail.